cg/src/domains/subjects/subjects.service.js

```diff
@@ -104,30 +100,6 @@ class SubjectsService {
     await this._saveSubjectEncryptionKey(trx, subjectId, encryptionKey);
   }
 
-  // This code is being used by the listeners in the processors domain. We should refactor this later to not have inter domain interaction
```

Comment:
I like that the code is going away. Still I'm considering how processors should act and what should they store to avoid duplication.

Reply:
Hmm I can't see a better way to do this. Maybe we could create a subjects service in a domain that's specific to those 2? Like a shared domain between them, with only 1 subjects service.

Reply:
I will bring it up in the next meeting, Monday